postreise/analyze/generation/summarize.py

```diff
@@ -157,18 +168,23 @@ def get_generation_time_series_by_resources(scenario, area, resources, area_type
     return pg.groupby(grid.plant.loc[plant_id, "type"].values, axis=1).sum()
 
 
-def get_storage_time_series(scenario, area, area_type=None):
+def get_storage_time_series(scenario, area, area_type=None, storage_e=False):
     """Get time series total storage energy in certain area of a scenario.
 
     :param powersimdata.scenario.scenario.Scenario scenario: scenario instance
     :param str area: one of *loadzone*, *state*, *state abbreviation*,
         *interconnect*, *'all'*
     :param str area_type: one of *'loadzone'*, *'state'*, *'state_abbr'*,
-    *'interconnect'*
+        *'interconnect'*
+    :param bool storage_e: if set, return time series energy of storage devices
+        instead of power generation. Default to False.
```

Comment:
The docstring doesn't seem to match the parameters. Docstring implies it's mainly energy, but the parameter that would make it energy defaults to False.

Reply:
I don't quite get it. Should we rename it into something like "get_storage_e" or "with_storage_e"?

Reply:
Sorry, I wasn't clear. I mean that the main text of the docstring says: `Get time series total storage energy`, but unless we specify an additional parameter it defaults to getting the power, not the energy.

Reply:
Done.